src/app/common/hooks/useFilterState.ts

```diff
@@ -22,14 +22,8 @@ export const useFilterState = <T>(
       if (filterCategory?.getItemValue) {
         itemValue = filterCategory.getItemValue(item);
       }
-      return values.every((filterValue) => {
-        if (!itemValue) return false;
-        const lowerCaseItemValue = String(itemValue).toLowerCase();
-        const lowerCaseFilterValue = String(filterValue).toLowerCase();
-        return lowerCaseItemValue.indexOf(lowerCaseFilterValue) !== -1;
-      });
+      return values.every((filterValue) => (itemValue ? itemValue.includes(filterValue) : false));
```

Comment:
Why this change? it was case-insensitive before due to the toLowerCase calls.

Reply:
Ah I see, is this to support the Select filter logic? If so we might want to have two different branches here depending on the filter type, so the Search ones can still be case-insensitive

Reply:
Ok I didn't know the requirement the filter has to be case insensitive. I'll restore it.

Reply:
I suppose it's not a hard requirement, I just thought it'd be nice. If it's difficult don't worry about it.

Reply:
Not hard, just oversimplification shortcut I shouldn't taken.